specification/ai/Face/models.common.tsp

```diff
@@ -294,3 +294,14 @@ model FaceError {
 }
 
 model FaceErrorResponse is Azure.Core.Foundations.ErrorResponseBase<FaceError>;
+
+alias BodyParameter<
+  T,
+  TName extends valueof string = "body",
+  TDoc extends valueof string = ""
```

Comment:
i suggest either give a default doc like "body parameter" or just remove it and add suppression. current template will result to every body has an empty description.